web-client/src/ustc-ui/Button/Button.tsx

```diff
@@ -31,7 +31,6 @@ export const Button = props => {
   const classes = classNames(
     className,
     'usa-button',
-    tooltip && 'usa-tooltip',
```

Comment:
`usa-tooltip` class is incompatible with `usa-button` now, since the former uses `inline-block` display styling and the latter uses `inline-flex`